.github/ISSUE_TEMPLATE/---document-issue-.md

```diff
@@ -0,0 +1,65 @@
+---
+name: 文档（Document Issue）
+about: 您可以提问文档相关的问题。 You could use this template for reporting an document issue.
+
+---
+
+非常感谢您提交关于飞桨文档的Issue，我们会认真听取您的意见，并进行改进。
+
+建立issue时，为快速解决问题，请您根据情况给出如下信息：
+- 标题：请包含关键词“XXX文档问题”，例如“add 文档问题” 或 ”paddle.add 文档问题“
+- 文档版本信息：请提供有问题的文档的版本号，例如 develop，1.8，2.0RC；
+
+### 文档问题描述：
+
+#### API文档描述是否清晰？
+如：文档描述看不懂，不知道这个API该什么用；文档公式错误；
+
+#### 参数说明是否清晰
+如：没有解释清楚这个参数的作用以及使用场景；参数说明有误；
+
+#### 返回/形状说明是否清晰
+如：返回说明错误；形状描述不清楚；
+
+#### 示例代码是否有效？
+如：没有示例代码；示例代码没有可指导性；示例代码跑不通；示例代码格式有问题；示例代码没有注释；
+
+#### 中英文内容是否一致？
+如：中英文API描述不一致；中英文API参数不一致；
+
+#### 其他
+如：文档页面打不开；文档缺失；文档中有死链；
+
+
+Thank you for submitting a PaddlePaddle documentation issue. We will listen to your opinions carefully and make improvements.
+
+In order to quickly solve your problem, when creating an issue, please provide the following information:
+**Document Information**
+- Title：Please include the keyword "XXX document issue", such as "add document issue" or "paddle.add document issue"
+- Doc Version：Please provide the version of the document, such as develop, 1.8, 2.0RC;
+
+### Document problem description：
+
+#### Documentat description is clear?
+For example: I don’t understand this document, I don’t know how to use this API; the document formula is wrong;
+
+#### Parameter description is clear?
+For example: the function and usage scenarios of this parameter are not explained clearly; the parameter description is incorrect;
+
+#### Return/Shape description is clear
+For example: return/shape description error; shape description is not clear;
+
+#### Sample code is clear?
+For example: there is no sample code; the sample code is not instructable; the sample code does not work; the sample code format is problematic; the sample code has no comments;
+
+#### Chinese content and English content is consistent?
+For example: Chinese and English API descriptions are inconsistent; Chinese and English API parameters are inconsistent;
```

Comment:
Chinese API in this doc is inconsistent with English API, including params, description, sample code, formula, etc.